packages/vue-query/src/useBaseQuery.ts

```diff
@@ -49,7 +49,7 @@ type UseQueryOptionsGeneric<
       TQueryFnData,
       TError,
       TData,
-      TQueryData,
+      TQueryFnData,
```

Comment:
I'm not sure if this is a proper change. Why this is needed?

Reply:
Oops! that's my fault, I'm already fix it